package.json

```diff
@@ -36,12 +38,13 @@
     "@semantic-release/release-notes-generator": "9.0.3",
     "codecov": "3.8.3",
     "config": "3.3.2",
+    "dotenv": "^16.4.5",
```

Comment:
Why do we need this new dependency?

Reply:
Adding dotenv was intended to facilitate local testing by allowing quick changes to environment variables, particularly when using real AWS keys and secrets. However, I understand it's not necessary and can be removed if preferred.

Reply:
Removed it